docs/get-started/_index.md

```diff
@@ -88,100 +83,42 @@ motor->set_power(0.4);
-Modules provide custom components or services as _modular resources_ which are run and managed by `viam-server` as processes.
-When configuring your machine, you can choose and configure built-in components and services, as well as any available from the Viam Registry.
-You can also create your own resources for any hardware or software and add them to the Viam Registry.
-
-### Better and smarter machines
-
-<div>
-<img src="data-ml.svg" alt="A rover detecting a flower" class="alignright" width="400px" >
-</div>
-
-The Viam app provides several higher-level functionalities to make your machines smarter and better, such as:
-
-- **Data Management**: You can collect data from any machine and automagically sync it to the cloud.
-  Intermittent connectivity is not a problem, data syncs when possible.
-  From there you can query it using SQL, MQL, or with code.
-- **Machine Learning**: You can use collected data to train machine learning models within the platform.
-  Once trained, you can deploy ML models to all your machines to enable them to intelligently see and interpret the world around them.
-  The models run locally and do not require internet access at scoring time.
-- **Simultaneous Localization And Mapping (SLAM)**: You can use machines to map their surroundings and to find their own positions on a map.
-
-{{< alert title="Example Applications" color="note" >}}
+{{% expand "Click here to see example projects you can build with this" %}}
 
 - Capture sensor data on 100s of boats and sync it to the cloud when machines have internet connectivity.
 
 - Use Machine Learning to detect wildlife and when detected start data capture and send alerts.
 
 - Allow delivery robots to use their location and SLAM to navigate intelligently between GPS coordinates.
-  {{< /alert >}}
 
-### Managing many machines
+{{% /expand%}}
 
-Viam allows you to manage not just individual machines but entire fleets of thousands of machines.
-The [Viam app](https://app.viam.com)'s **fleet management** capabilities enable you to:
+### Fleet management
 
-- Manage software across your fleet, including deployment of code and machine learning models.
-- Update software, Configure, control, debug, and manage groups of machines in one go.
-- Manage permissions within your organization and locations.
+The [Viam app](https://app.viam.com)'s **fleet management** capabilities enable you to:
 
-You can collaborate on your machines and manage permissions for your fleet using **Role-Based Access Control** (RBAC).
-Users can have access to different fleet management capabilities depending on whether they are an owner or an operator of a given organization, location, or machine.
+- **Deployment**: Configure and update hardware, software, and machine learning models for groups of machines in one go.
+- **Management** Manage deployment and permissions using RBAC.
+- **Observability & Teleoperation**: Control and debug machines.
```

Comment:
```suggestion
- **Deploy**: Configure and update hardware, software, and machine learning models for groups of machines in one go.
- **Manage**: Manage deployment and permissions using RBAC.
- **Observe & Teleoperate**: Control and debug machines from anywhere.
```
grammatically wonky with "enable you to". Also, "Control and debug machines" alone sounds too short, and we've already talked about controlling and debugging earlier in the page, so I think the from anywhere part is the lead here.